room_assistance/indico_room_assistance/controllers.py

```diff
@@ -0,0 +1,63 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2019 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+
+import dateutil.parser
+from flask import request, session
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.orm import joinedload
+from werkzeug.exceptions import Forbidden
+
+from indico.core.db import db
+from indico.modules.events.requests.models.requests import Request, RequestState
+from indico.util.date_time import as_utc, get_day_end, get_day_start
+from indico.util.struct.iterables import group_list
+from indico.web.rh import RHProtected
+
+from indico_room_assistance.forms import RequestListFilterForm
+from indico_room_assistance.util import is_room_assistance_support
+from indico_room_assistance.views import WPRoomAssistance
+
+
+def _find_requests(from_dt=None, to_dt=None):
+    query = (Request.query
+             .options(joinedload(Request.event))
+             .filter(Request.type == 'room-assistance',
+                     Request.state == RequestState.accepted,
+                     db.cast(Request.data, postgresql.JSONB).op('?')('start_dt')))
```

Comment:
`db.cast(Request.data, postgresql.JSONB).has_key('start_dt')` should work as well.

BTW, what about converting the column to JSONB in the core?

Reply:
Hm, I thought I changed all the occurrences of that.